helm/hpcc/values.schema.json

```diff
@@ -1825,8 +1825,8 @@
           "description": "Should the index node memory allocation flush the cache and retry if memory allocation fails"
         },
         "fieldTranslationEnabled": {
-          "default": "payload",
-          "enum": ["false", "true", "payload"],
+          "default": "PayloadRemoveOnly",
+          "enum": ["false", "true", "PayloadRemoveOnly"],
```

Comment:
payload still needs to be in the list as a valid option.  It would be better to have consistent casing with the other options - a leading lower case letter.